### PR TITLE
Copy code snippet button

### PR DESCRIPTION
**Pull request by abrardev99**
This PR adds a copy button so people can easily copy a snippet. The original code comes from Laravel documentation [PR](https://github.com/laravel/laravel.com/pull/230).

Here is how it looks. 

https://user-images.githubusercontent.com/54532330/181061649-c6d1e698-7247-41ad-91cc-c1bc853a1099.mov

> Note: This does not add a copy button on snippets other than documentation like the home page. 

image

### CODE REVIEW

**Comment by stancl** [comment]
The PR looks great! Thanks a lot!

The only change I'm thinking about making is aligning the button with the first line in the code block:
CleanShot 2022-07-26 at 7 09 58@2x

You can probably just do this manually, position it a few pixels down so that the middle of the button is aligned with the middle of the first line.

That way, the button will be in the middle of the code block for single-line code blocks, and aligned with the first line in multi-line code blocks.

**Comment by abrardev99** [comment]
Adjusted the copy button a little bit.
Screenshot 2022-07-27 at 5 48 57 PM

Multiple line codeblock
Screenshot 2022-07-27 at 5 47 58 PM

File: source/_assets/css/clipboard.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .code-block-wrapper .copyBtn {
-    @apply absolute right-2 top-2 outline-none text-gray-200 opacity-25;
+    @apply absolute right-2 top-4 outline-none text-gray-200 opacity-25;
```
Comment (stancl) [inline source/_assets/css/clipboard.css:2]: Make it 5 or 6, so that the middle of the button is aligned with the text

**Comment by stancl** [comment]
To resolve the conflicts, you can just merge `master` into your branch, start conflict resolution, and re-run npm to override the file with conflicts.

File: source/_assets/css/clipboard.css (hunk 1 above)
Comment (abrardev99) [reply source/_assets/css/clipboard.css:2]: 5 looks perfect. 
image

**Comment by abrardev99** [comment]
>  you can just merge master into your branch, start conflict resolution, and re-run npm to override the file with conflicts.

An excellent way of resolving conflicts of lock files.

File: source/_assets/css/clipboard.css (hunk 1 above)
Comment (stancl) [reply source/_assets/css/clipboard.css]: Hmm, 5 seems *slightly* bit too low, but 4 was definitely too high, so I think we'll just go with 5 👍🏻

**Comment by stancl** [comment]
Thanks a lot for the PR!

**Pull request merged**